clang/runtime/dpct-rt/include/util.hpp

```diff
@@ -358,6 +358,35 @@ unsigned int match_all_over_sub_group(sycl::sub_group g, unsigned member_mask,
 }
 
 namespace experimental {
+
+#define CUDA_SHFL_SYNC(RES, MASK, VAL, SHFL_PARAM, C, SHUFFLE_INSTR)           \
```

Comment:
Please protect the code with maco __NVPTX__,  keep it only visible for CUDA backend.